src/PoolSelector.ts

```diff
@@ -99,10 +99,11 @@ export default class PoolSelector {
             return pool;
         }
 
-        const meetsMinSpotRequirement = this.maxMembers - poolInfo.memberCounter >= this.minSpots;
```

Comment:
@jonathanpdunne this check ensures that we only get pools that have a certain amount of free spaces. We can eliminate the call to check the max members for pools by hardcoding but we will still want to check if we have the min desired free spots in a pool.

Reply:
But since there is no limit on the number of spots in a pool, there will always be an *infinite* number of free spots, right?